web/packages/teleport/src/Account/ChangePassword/ChangePassword.tsx

```diff
@@ -43,12 +45,14 @@ export function ChangePassword({
       <FeatureHeader border="none">
         <FeatureHeaderTitle>Change Password</FeatureHeaderTitle>
       </FeatureHeader>
-      <PasswordForm
-        auth2faType={auth2faType}
-        preferredMfaType={preferredMfaType}
-        onChangePass={changePassword}
-        onChangePassWithWebauthn={changePasswordWithWebauthn}
-      />
+      <Card as="form" width="456px" p="6">
```

Comment:
i think because of the `form` there is some kind of form warning on the console log
<img width="1099" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/530fb8b0-a695-4b93-b340-220a7466f326">

Reply:
Strangely, it doesn't complain on my side. But I'll add it back anyway.